SConstruct

```diff
@@ -43,7 +43,7 @@ def get_version(env):
 
 
 API_VERSION_MAJOR = "2"
-API_VERSION_MINOR = "3"
+API_VERSION_MINOR = "4"
```

Comment:
https://github.com/daos-stack/daos/pull/9985 also changes this line and if the two PRs are merged then the change to the second one will be lost, so whichever PR lands seconds will need to rebase onto master after the first has landed and update this value to 5.

Reply:
since both updates are going on the same "release" and almost at the same time, there is no reason to bump this twice IMO

Reply:
anyway this is ready to land now, no?

Reply:
hopefully https://github.com/daos-stack/daos/pull/9985 will not need a rebase if this lands first (and if we decide to keep the same minor version value). That PR just got rebased for another issue & retested. At the moment it does need a +1 before requesting landing.

Reply:
Since this PR need be rebased due to conflict, i removed my API version bump and keep Ken's PR there.